modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterInitializer.java

```diff
@@ -122,44 +136,57 @@ public CompletableFuture<Void> initCluster(
 
             LOG.info("Resolved CMG nodes[nodes={}]", cmgNodes);
 
-            CmgInitMessage initMessage = msgFactory.cmgInitMessage()
-                    .metaStorageNodes(msNodeNameSet)
-                    .cmgNodes(cmgNodeNameSet)
-                    .clusterName(clusterName)
-                    .clusterConfigurationToApply(clusterConfiguration)
-                    .build();
-
-            return invokeMessage(cmgNodes, initMessage)
-                    .handle((v, e) -> {
-                        if (e == null) {
-                            LOG.info(
-                                    "Cluster initialized [clusterName={}, cmgNodes={}, msNodes={}]",
-                                    initMessage.clusterName(),
-                                    initMessage.cmgNodes(),
-                                    initMessage.metaStorageNodes()
-                            );
-
-                            return CompletableFuture.<Void>completedFuture(null);
-                        } else {
-                            if (e instanceof CompletionException) {
-                                e = e.getCause();
-                            }
+            String initialClusterConfiguration = clusterConfiguration == null
+                    ? ""
+                    : clusterConfiguration;
```

Comment:
I find it strange that you must convert null to empty string, just to call a private method, that does the same null-check. Something's wrong here, please simplify your code.